packages/nodejs/src/config.ts

```diff
@@ -89,7 +119,7 @@ export class Configuration {
       ignoreErrors: [],
       ignoreNamespaces: [],
       log: "file",
-      logPath: "/tmp",
+      logPath: this._tmpdir(),
```

Comment:
I think this should have no default now that I take a closer look. I'll address this in another PR if needed after taking a better look.